demo/angularjs/components/progress/doc.md

```diff
@@ -0,0 +1,13 @@
+# Progress
+
+**Displays a loading state.**
+
+There is two types of progress to differentiate loading activities. Only one type should represent each kind of activities.
```

Comment:
plusieurs -> pluriel -> `There are`

Même idée que pour les `chips` -> `The same indicator should always represent the same kind of activities`